src/base-commands/twilio-client-command.js

```diff
@@ -2,54 +2,72 @@ const chalk = require('chalk');
 const { flags } = require('@oclif/command');
 const BaseCommand = require('./base-command');
 const CLIRequestClient = require('../services/cli-http-client');
```

Comment:
nit: CLIRequestClient -> CliRequestClient for consistency, since we do TwilioCliError.

Reply:
Outside the scope of this review, but works for me. I propose we treat abbreviations and acronyms like lower-cased words when it comes to `camelCase` and `PascalCase` ([source](https://stackoverflow.com/a/27172000)).